docs/tutorials/hackathon-prep-course/8-managing-canisters.md

```diff
@@ -227,6 +228,16 @@ Balance: 3_100_000_000_000 Cycles
 Module hash: 0xf8680eb74022a1079012b7e9c644d1156580002a6126305791811533d3fd6f17
 ```
 
+The values returned in this output refer to the following:
+
+- Status: The canister's current status, such as `Running` or `Stopped`. 
```

Comment:
there is a third state `Stopping`
https://github.com/dfinity/interface-spec/blob/master/spec/index.md#canister-lifecycle-canister-lifecycle

Reply:
maybe we can link the docs here as well

Reply:
Added `Stopping` and the docs link.